docs/build/configure-did.md

```diff
@@ -1,56 +0,0 @@
-# Configure your DID
```

Comment:
The authentication page has references to this configure a did page, and I don't see any changes to the authentication page in this PR, so I assume it's leaving some dead links currently

Reply:
removed the dead link. there was one. good catch